src/api-reference/request/v4.request.markdown

```diff
@@ -355,16 +383,18 @@ Name|Type|Format|Description
 code|string|-|The code for the approval status the Request. <br>Possible values: `NOT_SUBMITTED`, `SUBMITTED`, `APPROVED`, `CANCELED` or `SENTBACK`
 name|string|-|The approval status of the Request in the current user's language
 
+[Back to top](#request-endpoints)
 
 #### <a name="customfield"></a>CustomField
 
 Name|Type|Format|Description
 ---|---|---|------
 code|string|-|The short code for the list item. For non-list fields, this value will be blank
-value|string|-|The value of the non-list item. For list fields, this value will be blank. <br>In case of an UPDATE of the Custom field, the value must either contain the text data for a non-list item or the unique id of the item (listItemId value) for a list item
-listItemId|string|-|The unique id of the item in case of a list item
+value|string|-|The value of the non-list item. For list fields, this is the unique id of the item
+listItemId|string|-|The unique id of the item in case of a list field
```

Comment:
To remove ?

Reply:
Not for the current PR, we'll discuss in more detail for a future documentation update.